gtest/src/manager.rs

```diff
@@ -516,7 +392,9 @@ impl ExtManager {
                 self.blocks_manager.get(),
             )
             .map_err(TestError::ReadStateError)
-        } else if let Some(mut program_mock) = actor.take_mock() {
+        } else if let Some(mut program_mock) =
+            Actors::modify(*program_id, |actor| actor.unwrap().take_mock())
```

Comment:
I'd use `expect` instead of `unwrap` and state that actors existence is checked upper within `executable_actor_data` proc.

Reply:
Fixed